scripts/merge-forward.sh

```diff
@@ -4,5 +4,9 @@
 set -exo pipefail
 
 git fetch --all
-git checkout -b v2-main origin/v2-main
-git merge origin/master --no-edit
\ No newline at end of file
+
+# create 'v2-main' if it does not already exist
+(git branch --list | grep 'v2-main') || git checkout -b v2-main origin/v2-main
```

Comment:
If I'm reading the man page right, this is exactly what '-B' is made to do. https://git-scm.com/docs/git-checkout

```suggestion
git checkout -B v2-main origin/v2-main
```

Reply:
You're right. I seem to have missed that in the man page.